src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/requirements.txt

```diff
@@ -51,8 +51,8 @@ aws-cdk.cx-api==1.137
 aws-cdk.region-info==1.137
 awscli==1.22.26
```

Comment:
the version of `awscli` is incomplatible to version of `botocore` like in above comment